open_rarity/scoring/scorer.py

```diff
@@ -33,9 +33,14 @@ def validate_collection(self, collection: Collection) -> None:
                 "numeric or date traits"
             )
 
-        if collection.token_standards != [TokenStandard.ERC721]:
+        allowed_standards = {
+            TokenStandard.ERC721,
+            TokenStandard.METAPLEX_NON_FUNGIBLE,
+        }
```

Comment:
FYI: Thought about putting it in a constant or env var but felt it wasn't worth given we are addressing 1155 soon

Reply:
Also didn't want people to do funky stuff like forcibly overwriting them :)